InvenTree/common/forms.py

```diff
@@ -176,22 +170,14 @@ def __init__(self, *args, **kwargs):
 
                     # Optional entries
                     elif col_guess in file_manager.OPTIONAL_HEADERS:
-                        # Set field name
-                        field_name = col_guess.lower() + '-' + str(row['index'])
                         # Get value
                         value = row.get(col_guess.lower(), '')
                         # Set field input box
-                        if 'price' in col_guess.lower():
-                            self.fields[field_name] = MoneyField(
-                                label=_(col_guess),
-                                default_currency=currency_code_default(),
-                                decimal_places=5,
-                                max_digits=19,
-                                required=False,
-                                default_amount=clean_decimal(value),
-                            )
-                        else:
-                            self.fields[field_name] = forms.CharField(
-                                required=False,
-                                initial=value,
-                            )
+                        self.fields[field_name] = forms.CharField(
+                            required=False,
+                            initial=value,
+                        )
+
+    def get_special_field(self, col_guess, row, file_manager):
```

Comment:
Shouldn't it return something by default, `None` maybe?
What would happen with `overriden_field` if it is not defined in subclass?

Reply:
pass just returns nothing so `bool(overriden_field) => False`